audience-analytics/index.rst

```diff
@@ -0,0 +1,289 @@
+.. _audience-tuto:
+
+Audience Analytics
+==================
+
+|
+
+Step 1 - Introduction
+*********************
+
+This documentation is meant at developers wanting to install, configure and launch Angus.ai audience analytics application on a personal computer.
+
+What data can be retrieved
+--------------------------
+
+.. image:: ../images/intro.png
+
+Angus.ai anonymous audience analytics solution computes (from each video stream) the following metrics:
+
+- The number of people passing by the camera/device,
```

Comment:
I think it is confusing, just camera is good enough no ?

Reply:
Done